CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.10.0
+
+- Replaced depreciated `Endpoint` with endpoint_url, disabled use of `Endpoint 2.0`
+- Updated `aws-config ` to 0.54.1
+- Updated `aws-sdk-athena` to 0.24.0
+- Updated `aws-sdk-s3` to 0.24.0
+- Updated `aws-sdk-sqs` to 0.24.0
+- Updated `aws-smithy-http` 0.54.4
+- Updated `aws-types` to 0.54.1
```

Comment:
**suggestion**: I don't expect that any consumers of the `cobalt-aws` will care about what specifically we changed in the internals of the code around endpoints, because it's all hidden inside the various `get_client` functions. However, there is a good chance that they may need to make similar changes if they're working with custom endpoints themselves. I would word this to highlight the possibility of breaking changes from these dependencies and give the reader an easy next step when trying to debug any issues, something like:

```suggestion
- Updated various AWS SDK dependencies. Note that the new versions include significant changes to endpoint URL resolution, which may introduce breaking changes for consumers; see the [`aws-sdk-rust` release notes](https://github.com/awslabs/aws-sdk-rust/releases/tag/release-2023-01-13) for details.
  - Updated `aws-config ` to 0.54.1
  - Updated `aws-sdk-athena` to 0.24.0
  - Updated `aws-sdk-s3` to 0.24.0
  - Updated `aws-sdk-sqs` to 0.24.0
  - Updated `aws-smithy-http` 0.54.4
  - Updated `aws-types` to 0.54.1
```